### ISSUE TITLE
SyntaxError: Void elements do not have end tags

### ISSUE THREAD

**Issue by blackcater** [prettier/prettier#5600]
**Prettier 1.15.3**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeeBbADgGwIbwB8AOlMTKgJJRYCuMABDAJ5ZwC8xI8AHjFw1x4wcABYQcAEzgAnTiEAUroHYjQLAqAgM4BLAF4cu+GQHM4AgG4BaDBGk55AMwgyMAEQJ4AdLXWyoeDHABZazgcLhIyfgpKSCgmVj0QNhl1aA0cCBh5LBk4NihJASQtXXkARgA2MNQAemjocPIa6joYcJrMISIQABoQCCwYTWh1ZFA8GRkIAHcABXGEEZQ8UwhNSR6QACMZYQBrOBgAZSxhTShDZBgZWjhe0RgMHAB1UU14dRORQ4W3zVM35jIcDqEa9M7eGQwGY7QwYPDIOx4HDeXoAK3UPAAQjswPsjn44AAZM5wBFIlEgdE8Q5nQw4OAARVoGVJSERyNuiXGEKBphuG2yZxgTzWMFEyAAHAAGXrZCDeJ47LBA7JwCGmUm9HIAR1omhy0LwsPhbPJnO8GE0l2u5tp9KZLLJHN6MDwmxFkjFyAATC6dpocLSAMIQDBwoFQaCakBeOAAFTdi3Z3gAvimgA)
```sh
--parser vue
```

**Input:**
```vue
<template>
	<Input type="text" placeholder="用户名" size="large" v-model="formData.usernameModel">
		<Icon type="person" slot="prepend" :size="16"></Icon>
	</Input>
</template>
```

**Output:**
```vue
SyntaxError: Void elements do not have end tags "Input" (4:2)
  2 | 	<Input type="text" placeholder="用户名" size="large" v-model="formData.usernameModel">
  3 | 		<Icon type="person" slot="prepend" :size="16"></Icon>
> 4 | 	</Input>
    | 	^
  5 | </template>
```

**Expected behavior:**

```vue
<template>
	<Input type="text" placeholder="用户名" size="large" v-model="formData.usernameModel">
		<Icon type="person" slot="prepend" :size="16"></Icon>
	</Input>
</template>
```

> I try the same code in prettier@1.11.1, it works.

**Comment by playwolsey** [comment]
[error] src/pages/dev/index.vue: SyntaxError: Void elements do not have end tags "Col" (10:19)
[error]    8 |       <div class="divider"></div>
[error]    9 |       <Row>
[error] > 10 |         <Col>other</Col>
[error]      |                   ^
[error]   11 |         <Col>other</Col>
[error]   12 |       </Row>
[error]   13 |     </div>

**Comment by playwolsey** [comment]
[error] src/pages/dev/index.vue: SyntaxError: Void elements do not have end tags "Col" (10:19)
[error]    8 |       <div class="divider"></div>
[error]    9 |       <Row>
[error] > 10 |         <Col>other</Col>
[error]      |                   ^
[error]   11 |         <Col>other</Col>
[error]   12 |       </Row>
[error]   13 |     </div>


it seems not worked

**Comment by ikatyang** [comment]
@playwolsey The fix will be available in the next release (1.16), or you can install the dev version from GitHub (`npm install prettier/prettier`) as a temporary workaround.

**Comment by dragon-yuan** [comment]
> @playwolsey The fix will be available in the next release (1.16), or you can install the dev version from GitHub (`npm install prettier/prettier`) as a temporary workaround.

Thanks

**Comment by ikatyang** [comment]
@balupton This issue is for `--parser vue` but it seems you're using `--parser html` (`index.html`), which is considered HTML5 and tag names are case-insensitive there. Could you open a new issue with more details (e.g., playground link, why are tag names case-sensitive there)?
